src/util/functions.js

```diff
@@ -1,5 +1,5 @@
 import distanceInWordsStrict from 'date-fns/distance_in_words_strict';
-import { CIVIC_API_KEY } from 'react-native-dotenv';
+const CIVIC_API_KEY = 'AIzaSyDJeJLI-mFC5zx1zXCptCNnvk4e3iPbQF8';
```

Comment:
Why?

Reply:
Maybe you or @gmferland knows how to add these secrets to CircleCI? I'm not sure, but I think this package is dependent on it having an actual .env file to pull variables from so not 100% sure that's the solution.

Reply:
Yeah that's on my to-do list for after the auth stuff is done. But I think CircleCI allows you to store secret variables that it imports as environment variables in the traditional sense (i.e. `process.env.ENVIRONMENT_VARIABLE`). Not sure if react-native-dotenv also supports variables defined that way

Reply:
Good point greg - that is how circleci accesses them in my understanding as well.  We could still use this package and have our .env file reference process.env (and even put in our dev db as a default, so that we could use it without having to all individually encode the prod one into our process.env, but still keep our prod keys secure